package.json

```diff
@@ -239,17 +244,20 @@
     "start-postgres": "bash dev/postgres/10/start.sh",
     "start-mssql": "bash dev/mssql/2019/start.sh",
     "start-db2": "bash dev/db2/11.5/start.sh",
+    "start-oracle": "bash dev/oracle/21-slim/start.sh",
```

Comment:
It would only work on Apple Silicon under Rosetta because the Oracle Instant Client is not available on Apple Silicon due to some 3rd party libraries not being available.